Makefile

```diff
@@ -13,6 +13,7 @@ test: build-all-scenarios
 		--ignore=./smarts/env/tests/test_benchmark.py \
 		--ignore=./smarts/env/tests/test_learning.py \
 		-k 'not test_long_determinism'
+	rm -f .coverage*
```

Comment:
Just to be a tad more careful... everywhere where we keep such an `rm` like this, we should change the thing being removed to `.coverage.*`.  (Notice the extra dot.)  The more restrictive we can make the pattern, the safer it is.